minecraft_servers/froschmc/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "froschmc",
+  "nice_name": "FROSCHMC",
+  "direct_ip": "froschmc.de",
+  "server_wildcards": [
+    "%.froschmc.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://www.froschmc.de",
+    "web_shop": "https://froschmc.de",
```

Comment:
Either it's a shop or the normal website but not both (and remove the other key respectively)